modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -16,6 +19,21 @@ class ApplicationController < ::OpenidApplicationController
 
     private
 
+    def format_evss_errors(errors)
+      errors.map do |error|
+        { status: 422, detail: "#{error['detail'] || error['text']} #{error['severity']}", source: error['key'] }
+      end
+    end
+
+    def track_evss_validation_errors(errors)
+      StatsD.increment STATSD_VALIDATION_FAIL_KEY
+
+      errors.each do |error|
+        key = error['key'].gsub(/\[(.*?)\]/, '')
+        StatsD.increment STATSD_VALIDATION_FAIL_TYPE_KEY, tags: ["key: #{key}"]
```

Comment:
Will this allow for granular enough mapping of the fail types, or should each fail-type have it's own StatsD counter? Do you have a PromQL example lined up for what you'd like to see?


Just curious not a PR blocker

Reply:
We've been persisting to see what kind of validation failures we get, just collecting the data for now.